packages/wallet/api/test/test-lib-wallet.js

```diff
@@ -1193,6 +1193,166 @@ test('lib-wallet addOffer for autoswap swap', async (/** @type {LibWalletTestCon
   );
 });
 
+test('lib-wallet performAction suggestIssuer', async (/** @type {LibWalletTestContext} */ t) => {
+  t.plan(1);
+  const { board, wallet } = await setupTest(t);
+  const { issuer: bucksIssuer } = makeIssuerKit('bucks');
+  const bucksIssuerBoardId = await E(board).getId(bucksIssuer);
+
+  const action = JSON.stringify({
+    type: 'suggestIssuer',
+    data: { petname: 'bucksIssuer', boardId: bucksIssuerBoardId },
+  });
+  const done = await wallet.performAction({ action });
+
+  assert(done);
```

Comment:
failure message would be more informative with:
```suggestion
  t.truthy(done);
```

Reply:
Done